client/viame-web-common/components/RunPipelineMenu.vue

```diff
@@ -89,90 +101,148 @@ export default defineComponent({
-              class="pl-1"
+  <div>
+    <v-menu
+      max-width="320"
+      offset-y
+    >
+      <template v-slot:activator="{ on: menuOn }">
+        <v-tooltip bottom>
+          <template #activator="{ on: tooltipOn }">
+            <v-btn
+              text
+              :small="small"
+              :disabled="pipelinesNotRunnable"
+              v-on="{ ...tooltipOn, ...menuOn }"
             >
-              Run pipeline
-            </span>
-          </v-btn>
-        </template>
-        <span>Run CV algorithm pipelines on this data</span>
-      </v-tooltip>
-    </template>
+              <v-icon color="accent">
+                mdi-pipe
+              </v-icon>
+              <span
+                v-show="!$vuetify.breakpoint.mdAndDown"
+                class="pl-1"
+              >
+                Run pipeline
+              </span>
+            </v-btn>
+          </template>
+          <span>Run CV algorithm pipelines on this data</span>
+        </v-tooltip>
+      </template>
 
-    <template>
-      <v-card v-if="pipelines">
-        <v-card-title>
-          VIAME Pipelines
-        </v-card-title>
+      <template>
+        <v-card v-if="pipelines">
+          <v-card-title>
+            VIAME Pipelines
+          </v-card-title>
 
-        <v-card-text class="pb-0">
-          Choose a pipeline type. Check the
-          <a
-            href="https://viame.github.io/VIAME-Web/Pipeline-Documentation/"
-            target="_blank"
-          >docs</a>
-          for more information about these options.
-        </v-card-text>
-        <v-row class="px-3">
-          <v-col
-            v-for="(pipeType) in Object.keys(pipelines)"
-            :key="pipeType"
-            cols="6"
-          >
-            <v-menu
+          <v-card-text class="pb-0">
+            Choose a pipeline type. Check the
+            <a
+              href="https://viame.github.io/VIAME-Web/Pipeline-Documentation/"
+              target="_blank"
+            >docs</a>
+            for more information about these options.
+          </v-card-text>
+          <v-row class="px-3">
+            <v-col
+              v-for="(pipeType) in Object.keys(pipelines)"
               :key="pipeType"
-              offset-y
+              cols="6"
             >
-              <template v-slot:activator="{ on }">
-                <v-btn
-                  depressed
-                  block
-                  v-on="on"
-                >
-                  {{ pipeTypeDisplay(pipeType) }}
-                  <v-icon
-                    left
-                    color="accent"
-                    class="ml-0"
+              <v-menu
+                :key="pipeType"
+                offset-y
+                max-height="80%"
+                allow-overflow
+              >
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    depressed
+                    block
+                    v-on="on"
```

Comment:
I have an alternative idea here I want to run it by you because there are probably things that I'm not properly considering.

https://user-images.githubusercontent.com/61746913/106922851-757e1400-66db-11eb-8960-cee470b441fb.mp4

Reply:
I accept your proposal, feel free to push it.